ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/qos_profiles.schema.yml

```
@@ -78,6 +78,64 @@ keys:
                     description: |
                       Supported options are platform dependent
                       Example: "< rate > kbps", "1-100 percent", "< rate > pps"
+              random_detect:
+                type: dict
+                keys:
+                  ecn:
+                    description: Explicit Congestion Notification
+                    type: dict
+                    keys:
+                      count:
+                        type: bool
+                        description: |
+                          Enable counter for random-detect ECNs
+                      threshold:
+                        type: dict
+                        keys:
+                          units:
+                            required: true
+                            type: str
+                            valid_values: ["segments", "bytes", "kbytes", "mbytes"]
+                            description: |
+                              Indicate the units to be used for the threshold values.
+                              This should be one of segments, byte, kbytes, mbytes.
+                          min:
+                            type: int
+                            required: true
+                            min: 1
+                            max: 256000000
+                            convert_types:
+                              - str
+                            description: |
+                              Set the random-detect ecn minimum-threshold
+                          max:
+                            type: int
+                            required: true
+                            min: 1
+                            max: 256000000
+                            convert_types:
+                              - str
+                            description: |
+                              Set the random-detect ecn maximum-threshold
+                          max_probability:
+                            type: int
+                            min: 1
+                            max: 100
+                            convert_types:
+                              - str
+                            description: |
+                              Set the random-detect ecn max-mark-probability.
+                              The value should be between 1 and 100.
```

Avoid the "set the ...". Also try to write a description that is more clear text and helping the user if the key itself is not self-explanatory.
If the description is =~ the key, just remove the description.
Also the min/max are written in the docs, so you don't have to repeat in the description.
```suggestion
                              Random-detect ECN maximum mark probability.
```